app/controllers/confirmations_controller.rb

```diff
@@ -0,0 +1,17 @@
+class ConfirmationsController < ApplicationController
+  def verify_email
+    return unless params[:id] && params[:token]
+
+    @user = User.find_by(id: params[:id])
+
+    debugger
+    if @user.verified?
+      redirect_to login_url, notice: t(".already_verified")
+    elsif @user.auth_token_valid?(params[:token])
+      @user.update(verified_at: Time.now)
```

Comment:
create a method on user model and update column there.

We should always create model level updation method in model